linera-service/tests/end_to_end_tests.rs

```diff
@@ -59,7 +63,7 @@ impl FungibleApp {
     async fn entries(&self) -> Vec<native_fungible::AccountEntry> {
         let query = "accounts { entries { key, value } }";
         let response_body = self.0.query(&query).await.unwrap();
-        serde_json::from_value(response_body["accounts"]["entries"].clone()).unwrap_or_default()
+        serde_json::from_value(response_body["accounts"]["entries"].clone()).unwrap()
```

Comment:
You mean to give a type hint: `serde_json::from_value::<Option<_>>(response_body["accounts"]["entries"].clone()).unwrap()`

Reply:
IIRC this should return an empty `Vec` when there are no entries, so we should be good here. If it throws, it's an actual error